code_examples/sdk_examples/src/dapp/index.ts

```diff
@@ -1,10 +1,10 @@
 import * as Kilt from '@kiltprotocol/sdk-js'
 import { main as attestCredential } from './dapp/attestCredential'
 import { createFullDid } from '../workshop/attester/generateDid'
-import { domainLinkageCType } from './dapp/domainLinkageCtype'
 import { main as formatCredential } from './dapp/formatCredential'
 import { generateAccount } from '../workshop/attester/generateAccount'
 import { generateKeypairs as generateAttesterKeypairs } from '../workshop/attester/generateKeypairs'
+import { main as getDomainLinkageCType } from './dapp/domainLinkageCtype'
 import { main as getDomainLinkageCredential } from './dapp/domainLinkageClaim'
```

Comment:
```suggestion
import { getDomainLinkageCType } from './dapp/domainLinkageCtype'
import { getDomainLinkageCredential } from './dapp/domainLinkageClaim'
```
renaming the functions in their own file.

Reply:
I know too little about our tooling handling the snippets to say if this won't break it